code/modules/recycling/disposal-construction.dm

```diff
@@ -59,15 +59,14 @@
 				base_state = "intake"
 				dpdir = dir
 
-			if(9)
+			if(9, 11)
 				base_state = "pipe-j1s"
 				dpdir = dir | right | flip
 
-			if(10)
+			if(10, 12)

```

Comment:
because disposal pipe construction code is awful shit and the defines don't match with the numbers here